recipes/aiida-core/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "aiida-core" %}
+{% set version = "0.12.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 482e7aff408d6cc21e8ddde22efaa124d71619de53eccd37174c8e6d7e7e141e
+
+build:
+  number: 0
+  entry_points:
+    - verdi=aiida.cmdline.verdilib:run
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  skip: True  # [py > 27]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - reentry ==1.2.2
```

Comment:
these exact pins will make your package very fragile.  Are they really necessary?  If not, please remove them.

Reply:
@msarahan Unfortunately, from experience, they are a necessary evil.

> these exact pins will make your package very fragile

I would argue the problem is actually the opposite - namely that `aiida-core` *is* very fragile and the exact pins prevent it from breaking.

Reply:
Okie dokie...  Please keep an eye out for any users who report unsatisfiable errors or unexpected downgrades of software.  If everyone in conda-forge did this, essentially nothing would work.

Reply:
Understood.
I'll be pushing to both reduce the number of dependencies and to relax at least some of the constraints.